lib/ansible/modules/files/copy.py

```diff
@@ -166,6 +167,14 @@
     mode: '0644'
     backup: yes
 
+- name: Recursively copy and assign 755 permissions to all contents(directories and files) from "etc" directory into "path" directory
+  copy:
+    src: /mine/
+    dest: /path/
+    owner: root
+    group: root
+    directory_mode: 0755
```

Comment:
Suggest altering to use quotes here.

```suggestion
    directory_mode: '0755'
```